apps/passport-client/components/screens/VerifyScreen.tsx

```diff
@@ -123,7 +123,7 @@ function getCard(result: VerifyResult) {
         <CardHeader col="var(--accent-lite)">
           VERIFIED ZUZALU PASSPORT
         </CardHeader>
-        <MainIdentityCard showQrCode={false} user={result.user} />
+        <MainIdentityCard user={result.user} />
```

Comment:
the main identity card never shows a qr code - it shows the funny randomly generated icon instead